.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,4 @@
-### Acceptance Criteria
-- Include here all things that this PR should solve
+# Choose a pull request template
 
-
-### Security Checklist
-- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.
+- [feature PR](?template=feature.md&quick_pull=1)
+- [Release process](https://github.com/HathorNetwork/hathor-wallet-lib/compare/release...master?template=release.md&quick_pull=1)
```

Comment:
In the content of this file, you wrote:

> This PR will merge any changes on `release-candidate` into `release` branch.

But this link is configured to compare the `release` and `master`branches.

I believe the link should be fixed, right?

Reply:
Besides, should we have another template for merging `master` into `release-candidate`?

Reply:
Good catch! Fixed on https://github.com/HathorNetwork/hathor-wallet-lib/pull/632/commits/999f57d50e830c7b2b75cad894a789f7f5c9e4e5